### PR TITLE
[STAT-430] update other section

### PR DESCRIPTION
**Pull request by dastrong**
### What does this PR do?

Updates the Other section onboarding to use our new static analyzer versus our old one.

### Motivation

Transitioning our suggestions from our old to new static analyzer now.

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/continuous_integration/static_analysis/_index.md [hunk 1]
```diff
@@ -95,26 +94,33 @@ Configure the following environment variables:
...
 
-| Name       | Description                                                                                                                | Required | Default         |
-|------------|----------------------------------------------------------------------------------------------------------------------------|----------|-----------------|
-| `service`  | The name of the service to tag the results with.                                                                           | Yes      |                 |
-| `env`      | The environment to tag the results with. `ci` is a helpful value for this input.                                           | No       | `none`          |
+<div class="alert alert-info">
+  Add a `--performance-statistics` flag to your static analysis command to get execution time statistics for analyzed files.
+</div>
 
 Add the following to your CI pipeline:
```
Comment (juli1) [inline content/en/continuous_integration/static_analysis/_index.md:107]: We should add a note that the following script works for `x86_64` on Linux. We should list all the architecture available and their link to to the latest download.

File: content/en/continuous_integration/static_analysis/_index.md [hunk 2]
```diff
@@ -95,26 +94,33 @@ Configure the following environment variables:
...
+# Download the latest Datadog static analyzer:
+# https://github.com/DataDog/datadog-static-analyzer/releases
+DATADOG_STATIC_ANALYZER_URL=https://github.com/DataDog/datadog-static-analyzer/releases/latest/download/datadog-static-analyzer-x86_64-unknown-linux-gnu.zip
+curl -L $DATADOG_STATIC_ANALYZER_URL > /tmp/datadog-static-analyzer
+unzip /tmp/datadog-static-analyzer
 
-# Run Static Analysis (requires a pre-installed JVM)
-/tmp/cli-1.0-SNAPSHOT/bin/cli --directory . -t true -o results.sarif -f sarif
+# Run Static Analysis
+/tmp/datadog-static-analyzer -i . -o /tmp/report.sarif -f sarif --cpus <cpu_count>
```
Comment (juli1) [inline content/en/continuous_integration/static_analysis/_index.md:120]: No eed to specify the cpu count here.

**Review by juli1 — CHANGES_REQUESTED** [review]

File: content/en/continuous_integration/static_analysis/_index.md [hunk 3]
```diff
@@ -97,29 +96,54 @@ Configure the following environment variables:
...
+
+Select an analyzer for your architecture and OS:
+
+| Architecture | OS        | Name                                                    |
+|--------------|-----------|---------------------------------------------------------|
+| `aarch64`    | `Darwin`  | `datadog-static-analyzer-aarch64-apple-darwin.zip`      |
+| `aarch64`    | `Linux`   | `datadog-static-analyzer-aarch64-unknown-linux-gnu.zip` |
+| `x86_64`     | `Darwin`  | `datadog-static-analyzer-x86_64-apple-darwin.zip`       |
+| `x86_64`     | `Linux`   | `datadog-static-analyzer-x86_64-pc-windows-msvc.zip`    |
+| `x86_64`     | `Windows` | `datadog-static-analyzer-x86_64-unknown-linux-gnu.zip`  |
```
Comment (juli1) [inline content/en/continuous_integration/static_analysis/_index.md:117]: @dastrong let's also have a link to the latest

**Review by juli1 — CHANGES_REQUESTED** [review]

File: content/en/continuous_integration/static_analysis/_index.md (hunk 3 above)
Comment (juli1) [reply content/en/continuous_integration/static_analysis/_index.md:117]: The user should not have to guess the link themselves. We should give the links to them.

Just have a fourth column with a link to the download.

Example of link: `https://github.com/DataDog/datadog-static-analyzer/releases/latest/download/datadog-static-analyzer-x86_64-unknown-linux-gnu.zip` - just replace `datadog-static-analyzer-x86_64-unknown-linux-gnu.zip`. It gives a link to the latest release if people want to integrate it in their CI.

**Review by rtrieu — APPROVED** [review]
LGTM. Please let us know via requesting re-review or in #documentation when you are ready to merge this!

**Comment by dastrong** [comment]
@rtrieu it's ready to go now

**Pull request merged**